Nodejs/Product/Nodejs/Intellisense/IntellisenseController.cs

```diff
@@ -648,8 +644,7 @@ internal sealed class IntellisenseController : IIntellisenseController, IOleComm
                         case VSConstants.VSStd2KCmdID.COMPLETEWORD:
                             ForceCompletions = true;
                             try {
-                                TriggerCompletionSession((VSConstants.VSStd2KCmdID)nCmdID == VSConstants.VSStd2KCmdID.COMPLETEWORD
-                                    && !NodejsPackage.Instance.IntellisenseOptionsPage.OnlyTabOrEnterToCommit);
+                                TriggerCompletionSession(false);
```

Comment:
`OnlyTabOrEnterToCommit` defaults to true in my understanding, which would make this `(VSConstants.VSStd2KCmdID)nCmdID == VSConstants.VSStd2KCmdID.COMPLETEWORD && !true`. Is that not correct?